tests/test_sample.py

```diff
@@ -201,6 +202,7 @@ def test_pauliz_hadamard_pauliy(self, theta, phi, varphi, device, shots, tol):
         ) / 4
         assert np.allclose(var, expected, **tol)
 
+    @flaky(max_runs=5, min_passes=3)
```

Comment:
Did the consistency of this test drop somehow with the new release of PL and qiskit? that's pretty weird no?

Reply:
I don't think so. Note that [we were pulling PennyLane v0.20.0](https://github.com/PennyLaneAI/pennylane-qiskit/runs/5100591876?check_suite_focus=true#step:5:28) when the CI check was failing and the sampling logic was not affected with the latest changes of PennyLane-Qiskit.

This test has been flaky in the past too, although usually, 1 re-run sufficed.

Reply:
Ahh okay,